### DISCUSSION TITLE
Development moves to a single new "main" branch

### DISCUSSION THREAD

**Discussion by ahojnnes** [colmap/colmap#2109] [General]
I decided to change the existing development model, where latest commits go into the "dev" branch and "master" points to the last official release commit. This has proven to be confusing to users and due to the "infrequent" release schedule, the commit in "master" was not really working on most machines anymore.

From today on, there is a single official "main" branch. The recommendation for all users is to consume from this branch or checkout a specific release tag, if necessary.

On your existing local clone of colmap, you will have to run:
```
git branch -m dev main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

@tsattler @mihaidusmanu @Skydes FYI

**Comment by fabianstahl** [comment]
It would be helpful to adapt the installation instructions in the docs accordingly to avoid confusion.

**Reply by ahojnnes** [reply]
I think that's already the case except for not yet regenerating the latest docs and publishing it on the website. Pleas let me know if there is still something inconsistent in the markdown source code.

**Reply by ahojnnes** [reply]
I just updated the web page, so colmap.github.io should show the same as the markdown source files.
